src/components/AdjustableSidebarWidth.vue

```diff
@@ -113,6 +114,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    disableResizing: {
+      type: Boolean,
+      default: false,
+    },
+    hardWidth: {
```

Comment:
This might just be me but "hard" width is a slightly non-obvious term. How about `fixedWith`? Just an idea, not a blocker.

Reply:
I like it.